test-project/PlaybackViewController.m

```diff
@@ -39,11 +40,34 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
   }
 }
 
+- (void) toggleFullscreen:(DMPlayerViewController *)player {
+
+    if (player.fullscreen) {
+        self.playerSize = self.initialPlayerSize;
+    }
+    else {
+        self.playerSize = self.view.frame.size;
+    }
+
+    self.playerWidthLayoutConstraint.constant = self.playerSize.width;
+    self.playerHeightLayoutConstraint.constant = self.playerSize.height;
```

Comment:
Well, it looks good enough to demonstrate the idea. 
It would definitely look better with an animation but my Objective-C skills are getting old and I didn't have much time to spend on this...  :sweat_smile: